tests/copybw.cpp

```diff
@@ -128,6 +130,25 @@ int main(int argc, char *argv[])
     cout << "selecting device " << dev_id << endl;
     ASSERTDRV(cuDeviceGet(&dev, dev_id));
 
+    #if CUDA_VERSION >= 11030
```

Comment:
can we refactor this code into an helper function used by all the tests?

Reply:
Done. Can I ask you to re-review?